src/app/parcels/parcelsTable/ParcelsPage.tsx

```diff
@@ -351,39 +351,37 @@ const ParcelsPage: React.FC<{}> = () => {
                             router.push("/parcels");
                         }}
                         headerId="expandedParcelDetailsModal"
+                        footer={
+                            <Centerer>
+                                <LinkButton link={`/parcels/edit/${selectedParcelId}`}>
+                                    Edit Parcel
+                                </LinkButton>
+                                {selectedClientDetails && (
+                                    <>
+                                        <LinkButton
+                                            link={`/clients?clientId=${selectedClientDetails.clientId}`}
+                                        >
+                                            See Client Details
+                                        </LinkButton>
+                                        <LinkButton
+                                            link={`/clients/edit/${selectedClientDetails.clientId}`}
+                                        >
+                                            Edit Client Details
+                                        </LinkButton>
+                                    </>
+                                )}
+                            </Centerer>
+                        }
                     >
                         <OutsideDiv>
                             <ContentDiv>
                                 <Suspense fallback={<ExpandedParcelDetailsFallback />}>
                                     <ExpandedParcelDetails parcelId={selectedParcelId} />
                                 </Suspense>
                             </ContentDiv>
```

Comment:
Why did this get moved?

Reply:
The buttons are currently in the bottom of the page but it should go in the footer so that it is accessible immediately without having to scroll all the way down